feature-libs/checkout/base/components/checkout-delivery-mode/checkout-delivery-mode.component.html

```diff
@@ -4,63 +4,78 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
 
 <ng-container
   *ngIf="
-    (supportedDeliveryModes$ | async)?.length && !(isUpdating$ | async);
+    !(isUpdating$ | async) &&
```

Comment:
Dom changes, need migration docs.

Please review the whole PR for dom changes and add corresponding migration doc.